Weather App/UI/HomeVC/HomeViewController.swift

```diff
@@ -29,20 +28,23 @@ class WeatherViewController: UIViewController {
     
     //MARK: Actions
     
-    private func setupCallbacks() {
-        
-    }
-    
     private func setupNavigationBar() {
+        self.title = "Home"
         self.navigationController?.navigationBar.setVisible(false)
         self.navigationController?.navigationBar.tintColor = .black
-        
+        removeTextFromBackButton()
         self.navigationItem.leftBarButtonItems = [
             UIBarButtonItem(image: UIImage(named: "search"), style: .plain, target: self, action: #selector(goToSearchScreen)),
```

Comment:
Based on the design, dont think the search and settings buttons are part of the navigation bar.